strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/api/impl/ldap/LdapAuthenticator.java

```diff
@@ -0,0 +1,29 @@
+package org.carlspring.strongbox.authentication.api.impl.ldap;
+
+import org.carlspring.strongbox.authentication.api.Authenticator;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.ldap.authentication.LdapAuthenticationProvider;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Component
+public class LdapAuthenticator
```

Comment:
> Also it will be clearer for users who wish to implement own Authenticators to use standard Spring components for this. 

They are able to implement their own Authenticators to use standard Spring components right now too. Please take a look at [How-To-Implement-A-Custom-Authenticator](https://github.com/strongbox/strongbox/wiki/How-To-Implement-A-Custom-Authenticator). They don't have to know where Authenticators are stored. They don't even have to know there is such thing as registry/manager.